hotline/urls.py

```diff
@@ -19,9 +21,24 @@
 
     # the django admin interface is always nice to have
     url(r'^admin/', include(admin.site.urls)),
-
     # the homepage goes straight to a template. But you may want to change this
     # into a normal view function
+    url(r'^$', TemplateView.as_view(template_name="home.html"), name='home'),
+    url(r'^adminpanel/?$', species.admin_panel, name='admin_panel'),
+    url(r'^adminpanel/species-list/?$', species.SpeciesList.as_view(), name='species_list'),
```

Comment:
Stick with the convention we've got going in all our ARC projects:

`url(r'^plural_object_name/page_type/?$', species.SpeciesList.as_view(), name='plural-object-name-page-type'),`

So the list page for species would be:
`url(r'^species/list/?$', species.SpeciesList.as_view(), name='species-list')`